src/site/design-elements/buttons/index.html

```diff
@@ -1,23 +1,27 @@
 <div id="buttons" class="headroom-large">
-  <h5>Buttons</h5>
+  <h2>Buttons</h2>
   <p>This is the primary button style for important calls to action. Two primary buttons cannot be used next to each other.</p>
   <div data-xrayhtml>
-    <button class="btn btn-mapzen">main call to action</button>
+    <button class="btn btn-darkpurple">primary button</button>
```

Comment:
yeah I do prefer your suggestion as well. The thing is that currently our product guide seems to expect specific color combination such as purple one + lighter background white one + darker background. (which seems to delude main/secondary functions for buttons)  so I ended up with naming this with color name in a hope that it will make it easy to follow these combinations. 

🤔 but it seems true that many buttons will be used on default, white background color so I am not against changing naming!